### DISCUSSION TITLE
How to get `refreshToken` when `signInWithEmailAndPassword` Android

### DISCUSSION THREAD

**Discussion by phuocantd** [invertase/react-native-firebase#7527] [Q&A]
In iOS, when I `signInWithEmailAndPassword` then I get `refreshToken` but Android is not .
How to get `refreshToken`. Thanks

**Comment by mikehardy** [comment]
https://stackoverflow.com/help/minimal-reproducible-example
https://stackoverflow.com/help/how-to-ask

**Reply by thisissmohan** [reply]
Can anyone please answer for above? i have the same question. if its not supported by native, then how i received it for IOS?
